arch/arm/core/cortex_m/fault.c

```diff
@@ -428,7 +429,7 @@ static int BusFault(z_arch_esf_t *esf, int fromHardFault, bool *recoverable)
 							((u32_t) &esf[0]));
 
 					if (min_stack_ptr) {
-						/* When BusFault Stacking Error
+						/* When bus_fault Stacking Error
```

Comment:
This needs to be BusFault - we are referring to the actual ARM Fault terminology